### PR TITLE
Allow editing of derivative on airplane

### PR DESCRIPTION
**Pull request by MrD-RC**
This configurator change allows derivative to be edited on the airplane platform. It is to work along side @avsaase PIDFF controller on airplanes. PR https://github.com/iNavFlight/inav/pull/6669

image

### CODE REVIEW

**Comment by avsaase** [comment]
Works like a charm. Thanks @MrD-RC!

File: js/fc.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ var FC = {
         return (MIXER_CONFIG.platformType == PLATFORM_AIRPLANE || MIXER_CONFIG.platformType == PLATFORM_ROVER || MIXER_CONFIG.platformType == PLATFORM_BOAT) || ((MIXER_CONFIG.platformType == PLATFORM_MULTIROTOR || MIXER_CONFIG.platformType == PLATFORM_TRICOPTER) && semver.gte(CONFIG.flightControllerVersion, "2.6.0"));
     },
     isRpyDComponentUsed: function () {
-        return MIXER_CONFIG.platformType == PLATFORM_MULTIROTOR || MIXER_CONFIG.platformType == PLATFORM_TRICOPTER;
+        return MIXER_CONFIG.platformType == PLATFORM_AIRPLANE || MIXER_CONFIG.platformType == PLATFORM_MULTIROTOR || MIXER_CONFIG.platformType == PLATFORM_TRICOPTER;
```
Comment (DzikuVx) [inline js/fc.js:74]: Actually, this contidion changes to always `true`. All platforms have D-term now

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: js/fc.js (hunk 1 above)
Comment (avsaase) [reply js/fc.js:74]: Also rovers?

Comment (DzikuVx) [reply js/fc.js:74]: @avsaase yes, rovers use FW controller

Comment (MrD-RC) [reply js/fc.js]: @DzikuVx updated

**Comment by avsaase** [comment]
On fixed wing it now shows Control Derivative instead of Feed Forward. I should have looked at your screenshot better...

**Comment by MrD-RC** [comment]
Shit, you’re right. I’ll see what’s going on here. It looks like those functions aren’t just used to check if D is needed or not.

**Comment by MrD-RC** [comment]
@avsaase Fixed #1217

**Pull request merged**